src/styles/list.scss

```diff
@@ -71,8 +71,13 @@
 .ob-list__text-tertiary,
 .ob-list__text-secondary {
   font-size: 0.8rem;
+}
+.ob-list__text-tertiary {
   color: #757575;
 }
+.ob-list__text-secondary {
+  color: grey;
```

Comment:
Should this be using one of the bulma colours?

Reply:
Idk, task said to change back to `grey` which is what it was before

Reply:
Wait, no it wasn't ir was $grey